.github/container/Dockerfile.pax.amd64

```diff
@@ -1,33 +1,46 @@
 # syntax=docker/dockerfile:1-labs
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax:mealkit
+ARG SRC_PATH_PAXML=/opt/paxml
+ARG SRC_PATH_PRAXIS=/opt/praxis
+
 ###############################################################################
-## Pax
+## Download source and add auxiliary scripts
 ###############################################################################
 
-ARG BASE_IMAGE=ghcr.io/nvidia/jax:latest
-FROM ${BASE_IMAGE}
+FROM ${BASE_IMAGE} as mealkit
+ARG SRC_PATH_PAXML
+ARG SRC_PATH_PRAXIS
 
-ADD install-pax.sh /usr/local/bin
-ADD install-flax.sh /usr/local/bin
-ADD install-te.sh /usr/local/bin
+# update TE manifest file to install the [test] extras
```

Comment:
Why here but not in Dockerfile.jax? TE is a part of JAX image now, so I would assume someone wants to test TE in JAX container.

Reply:
I don't mind. Any reason you kept it separate @yhtang ?

Reply:
So I took a look at it, and it appears we could merge the two without any issue since there's just a `sed` in [Dockerfile.pax.amd64](https://github.com/NVIDIA/JAX-Toolbox/pull/405/files#diff-249510ffdb78ebd74d306e940904393c34b2d8d580d2492a5cd60c58867cb5f4R16) that replaces the line, so I think it'd be fine to do that in requirements-jax.in

Any objections? @DwarKapex @yhtang 

Actually, another issue I found looking at this is that Dockerfile.pax.arm64 doesn't include this `sed`. Is that intentional @yhtang ?